docs-next/pages/guides/keystone-5-vs-keystone-next.mdx

```diff
@@ -0,0 +1,111 @@
+import { Page, components } from '../../components/Page';
+import { getHeadings } from '../../lib/getHeadings';
+import { InlineCode } from '../../components/Code';
+import { Alert } from '../../components/Alert';
+import { MDXProvider } from '@mdx-js/react';
+import Image from 'next/image';
+import Link from 'next/link';
+
+# Keystone 5 vs Next. Which should you use?
+
+<Alert>
+  Updated April 1st 30th. See our <Link href="https://next.keystonejs.com/roadmap">roadmap</Link> for the latest improvements.
+</Alert>
+
+Keystone 5 is now in **maintenance mode**. We‘re focusing all our efforts on Keystone Next, to be released as Keystone 6 later this year. If you’re wondering which version to start your next project with, this guide is for you.
+
+---
+
+**TL;DR:** Both versions are production ready. Use Keystone 5 if you need a stable, battle-tested foundation to build on today. If you’re happy to manage a few updates over the coming months then Keystone Next is what you want. 
+
+
+## What’s changing between Keystone 5 and 6
+
+There are a few major things that will change between 5 and 6 which we’re currently implementing in Keystone Next:
+
+- New field types. Including a new [Document field](../guides/document-fields) field which replaces the old Content field.
+- New [configuration syntax](../apis/config) and [CLI](../guides/cli).
+- New express-independent session system.
+- New internal APIs for querying lists.
+- Completely new Admin UI.
+
+### DB adapters out. Prisma in.
+
+We’re also moving away from maintaining several database adapters to just using **Prisma**, which simplifies things. Until Prisma support MongoDB (which is planned) we’ll have a gap in our database support.
```

Comment:
```suggestion
We’re also moving away from maintaining several database adapters to just using **Prisma**, which simplifies things. Until Prisma support MongoDB (which is [planned](https://www.notion.so/Prisma-Roadmap-50766227b779464ab98899accb98295f)) we’ll have a gap in our database support.
```

Reply:
this is great. i went looking for something like this link but found an old GH issue with lots of people complaining. wasn't the best optics.